git_machete/client.py

```diff
@@ -2084,7 +2098,8 @@ def __handle_untracked_state(
                 opt_yes=opt_yes)
         elif len(rems) == 1:
             self.handle_untracked_branch(
-                new_remote=rems[0], branch=branch,
+                new_remote=rems[0],
+                branch=branch,
                 is_called_from_traverse=is_called_from_traverse,
```

Comment:
...for instance, here `is_called_from_create_pr` is silently assumed to be `False`... should it be tho? 🤔